README.md

```diff
@@ -469,10 +470,13 @@ This repository contains a curated list of awesome open source libraries that wi
 * [Haystack](https://github.com/deepset-ai/haystack) ![](https://img.shields.io/github/stars/deepset-ai/haystack.svg?style=social) - Haystack is an open source NLP framework to interact with your data using Transformer models and LLMs (GPT-3 and alike). Haystack offers production-ready tools to quickly build ChatGPT-like question answering, semantic search, text generation, and more.
 * [Kashgari](https://github.com/BrikerMan/Kashgari) ![](https://img.shields.io/github/stars/BrikerMan/Kashgari.svg?style=social) - Kashgari is a simple and powerful NLP Transfer learning framework, build a state-of-art model in 5 minutes for named entity recognition (NER), part-of-speech tagging (PoS), and text classification tasks.
 * [LangChain](https://github.com/hwchase17/langchain) ![](https://img.shields.io/github/stars/hwchase17/langchain.svg?style=social) - LangChain assists in building applications with LLMs through composability.
+* [NLTK](https://github.com/nltk/nltk) ![](https://img.shields.io/github/stars/nltk/nltk.svg?style=social) - NLTK (Natural Language Toolkit) is a suite of open source Python modules, data sets, and tutorials supporting research and development in Natural Language Processing.
```

Comment:
NLTK is a great library but more of a library than productionisation relevant framework

Reply:
> NLTK is a great library but more of a library than productionisation relevant framework

I saw a lot of NLP teams still take advantage of NLTK. Though it is losing its share compared to other tools such as Spacy, it is still impactful as a powerful NLP tool. @axsaucedo

Reply:
Yes NLTK is a good general library for NLP  but more of a library than productionisation relevant framework

Reply:
> Yes NLTK is a good general library for NLP but more of a library than productionisation relevant framework

What does productionisation mean in your mind? I mean, if this general tool belongs to machine learning category and contribute to real-world prodcuts, does it count as a productionisation tool? In this case, NLTK is a productionisation tool. @axsaucedo Maybe we can open up an issue for it.

Reply:
The way to think about is that this list should have MLOps tools that are used to productionise, not tools that are present in production because they are used primarily for experimentation and are deployed/productionised - this would be part of this issue https://github.com/EthicalML/awesome-production-machine-learning/issues/367, but I still don't see as part of this list, which is why removing for now

Reply:
If this tool helps with the deployment of NLP products then it should be included in the list. After viewing the tools in the list, the scope seems not narrowing down to exclusively the deployment/post-deployment things only. For example, there exist a lot of tools that focus most on experimentation things such as Mlflow, ModelDB, Jax, etc. If we do not intend to include tools that helps with ML production, then we should remove more than half od the tools in our list. @axsaucedo